docker/cygnus-ngsi/Dockerfile

```diff
@@ -16,7 +16,7 @@
 # For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
 #
 
-FROM centos:centos7.9.2009
+FROM centos:centos8.4.2105
```

Comment:
Suggested line for CNR:

```
[cygnus-common][cygnus-ngsi][cygnus-ngsi-ld][cygnus-twitter] Upgrade Dockerfile base image from centos7.9.2009 to centos8.4.2105
```
```

Reply:
Done in https://github.com/telefonicaid/fiware-cygnus/pull/2109/commits/a501b91df64c751b419e6521987489f8745b25d0